app/uk/gov/hmrc/voapropertylinking/repositories/DVRRepository.scala

```diff
@@ -87,10 +93,11 @@ case class DVRRecord(
       organisationId: Long,
       assessmentRef: Long,
       agents: Option[List[Long]],
-      createdAt: Long
+      createdAt: Option[BSONDateTime]
```

Comment:
if we make it optional, won't that lead to invalid JSON on the consumer side? I would have expected `createdAt` to be always present.

Reply:
this is how it was in prod